scripts/tasks/webpack.js

```diff
@@ -6,13 +6,16 @@ const path = require('path');
 const fs = require('fs');
 
 exports.webpack = webpackTask();
-exports.webpackDevServer = function() {
+exports.webpackDevServer = async function() {
+  const fp = require('find-free-port');
   const webpackConfigFilePath = argv().webpackConfig || 'webpack.serve.config.js';
   const configPath = path.resolve(process.cwd(), webpackConfigFilePath);
+  const port = await fp(4322, 4400);
 
   if (fs.existsSync(configPath)) {
+    ``;
```

Comment:
What's this line?

Reply:
dunno! weird. I removed it :) good catch